### DISCUSSION TITLE
Hari Golfianto | golf

### DISCUSSION THREAD

**Discussion by golfiantos** [onlyphantom/llm-python#31] [Practicum: Generative AI]
Hi Sam, nice to meet you. I appreciate the excellent teaching and the time you invested 🙌 Kindly check and review my code
Thank you

https://colab.research.google.com/drive/1GbgJvu8c1IlBKJmEGFGfDImX6tv_XTV6?usp=sharing

**Comment by onlyphantom** [comment]
Hey Hari! Interesting work on creating a separate `get_first_of_current_month` function! 

Also impressive that you've done both Challenge 1 and 2 as well! 

Marked as passed and will be issuing a certificate soon!
